src/immutable_data.rs

```diff
@@ -22,18 +22,20 @@ use xor_name::XorName;
 use sodiumoxide::crypto;
 
 #[derive(Hash, Clone, Eq, PartialEq, Ord, PartialOrd, RustcEncodable, RustcDecodable, Debug)]
-/// ImmutableDataType.
+/// The type of an individual copy of immutable data.
```

Comment:
Self validating data - i.e. name = Hash(content) or for backup name = Hash(Hash(content) etc.